RoboTownMechanics/Assets/Scripts/Menu/PauseSystem.cs

```diff
@@ -0,0 +1,30 @@
+using UnityEngine;
+
+
+namespace PauseHandler
+{
+    public class PauseSystem : SingletonBehaviour<PauseSystem>
+    {
+        //--------------------Public--------------------//
+        public bool isPaused;
```

Comment:
isPaused -> IsPaused. Forgot CamelCase

Reply:
I have changed said code to the conventions